packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html

```diff
@@ -2,15 +2,15 @@
   <div class="slds-grid feedback__main-section">
     <span class="slds-var-p-right_medium slds-text-color_weak feedback__question"
       id="feedback__question">{question}</span>
-    <button aria-label={likeLabel} data-cy="feedback__like-button" aria-describedby="feedback__question" onclick={handleLike}
+    <button aria-label={likeLabel} data-cy="feedback__like-button" title={labels.thisAnswerWasHelpful} aria-describedby="feedback__question" onclick={handleLike}
       class={likeButtonClass}>
       <lightning-icon aria-hidden="true" class={likeIconClass} icon-name={likeIconName} size={size}
         variant={likeIconVariant}></lightning-icon>
       <template if:false={hideLabels}>
         <span class="feedback__button-label slds-var-m-left_xx-small">{likeLabel}</span>
       </template>
     </button>
-    <button aria-label={dislikeLabel} data-cy="feedback__dislike-button" aria-describedby="feedback__question" onclick={handleDislike}
+    <button aria-label={dislikeLabel} data-cy="feedback__dislike-button" title={labels.thisAnswerWasNotHelpful} aria-describedby="feedback__question" onclick={handleDislike}
```

Comment:
```suggestion
    <button aria-label={dislikeLabel} data-cy="feedback__dislike-button" title={dislikeLabel} aria-describedby="feedback__question" onclick={handleDislike}
```

Reply:
You have to change the `title={dislikeLabel}` just like Mehdi proposed it, and it's the parent component in the quanticGenAI that should be setting the label correctly.